iceoryx_posh/include/iceoryx_posh/internal/popo/ports/client_server_port_types.hpp

```diff
@@ -67,6 +68,26 @@ using ClientChunkSenderData_t = ChunkSenderData<MAX_REQUESTS_ALLOCATED_SIMULTANE
 
 using ServerChunkSenderData_t = ChunkSenderData<MAX_RESPONSES_ALLOCATED_SIMULTANEOUSLY, ServerChunkDistributorData_t>;
 
+enum class ClientEvent : EventEnumIdentifier
+{
+    RESPONSE_RECEIVED
+};
+
+enum class ClientState : StateEnumIdentifier
+{
+    HAS_RESPONSE
```

Comment:
Maybe nitpicking, could be several ones HAS_RESPONES and HAS_REQUESTS would be the alternative. For the subscriber it is HAS_DATA, but DATA has no plural, your decision

Reply:
I don't have strong feelings but somehow like `HAS_RESPONES` ;)

There would also be the option to unify `SubscriberState`, `ClientState` and `ServerState` as
```cpp
enum class ConsumerState {
    HAS_DATA
};
```
and
```cpp
enum class ConsumerEvent {
    DATA_RECEIVED
};
```

@elfenpiff what do you think?

Reply:
@elBoberido My feeling is to either change it now or never. Breaking the API because of this in the next release is not worth it. What do you suggest?

Reply:
@budrus for the C++ API it is not user facing ... at least as far as I know. I'm not sure about the C API but I would keep it as is and if there is time before the release we could adjust it if @elfenpiff is in favor of it.

Reply:
This is already implemented for the C binding in some other PR, so if this changes the c binding has to be adapted but I do not have a problem with this.